cmd/geth/main.go

```diff
@@ -204,6 +204,8 @@ var (
 	metricsFlags = []cli.Flag{
 		utils.MetricsEnabledFlag,
 		utils.MetricsEnabledExpensiveFlag,
+		utils.MetricsEnabledRecordIOFlag,
+		utils.MetricsDisablePrefetchFlag,
```

Comment:
`utils.MetricsDisablePrefetchFlag` useless now, please remove it

Reply:
`MetricsEnabledRecordIOFlag` useless now, please remove it

Reply:
updated